libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts

```diff
@@ -16,7 +16,70 @@ const view: View = {
         columns: ["1fr"],
       },
     },
-    childrenKeys: ["importContactsButton", "contactsLoader"],
+    childrenKeys: ["contactsPanel", "contactsLoader"],
+  },
+  contactsPanel: {
+    component: "block-box",
+    childrenKeys: ["contactsSearch", "contactsButtonActions"],
+    layout: {
+      gridLayout: {
+        rows: [],
+        columns: ["280px", "344px"],
+        justifyContent: "space-between",
+      },
+    },
+  },
+  contactsSearch: {
+    component: "form",
+    config: {
+      formOptions: {
+        defaultValues: {
+          searchedContact: undefined,
+        },
+      },
+    },
+    childrenKeys: ["contactsSearchWrapper"],
```

Comment:
`contactsSearchWrapper` is not necessary, can be replaced by `contactsSearchInput`

Reply:
https://github.com/mudita/mudita-center/pull/2078/commits/ff1765b598b7425aad42b18003c5cdf4603df84f